rfcs/0000-rfc-template.md

```diff
@@ -22,6 +23,37 @@ Stage X: Provide a brief explanation of why the proposal is being marked as aban
 -->
 
 ## Fields
+The `vulnerabilities` fields being proposed are as follows:
+
+Field | Type | Description /Use Case
+vulnerability.vendor.id | keyword | A vulnerability doesn't have necessary a CVE associated with it. I don't know if it makes sense to seperate vulnerability ID (like CVEs) to the vendor/detection IDs.
```

Comment:
The CVE identifier is just an example for `vulnerability.id` in the field description today. Any unique ID can be used including one provided by a vendor.

Reply:
I like the seperation of fields`vulnerability.id` and `vulnerability.vendor.id`. It clarifies where data is located. And as CVE IDs is the unique common identifier between vulnerabilities, it makes sense to me.

Reply:
can we have both CVE and vendor id?

Reply:
@trisch-me CVE is already covered/mapped in existing ECS field i.e.,` vulnerability.id ` and my proposal is to add `vulnerability.vendor.id`

Reply:
yes, my question was about if one log entry can have both CVE and vendor.id?

Reply:
this question is still open